cmd/habitat/community/manager.go

```diff
@@ -37,6 +39,7 @@ func NewManager(path string, proxyRules *proxy.RuleSet) (*Manager, error) {
 			// TODO: @arushibandi remove this usage of compass
 			StartCmd: filepath.Join(compass.ProcsPath(), "bin", "amd64-darwin", "start-ipfs"),
 		},
+		p2pHost:        host,
```

Comment:
is there a way to abstract this away from community manager and only have it at the raft level?

Reply:
I think its a good idea for the host to be exposed at a higher level, because it allows us to multiplex connections between two nodes. That way if two different protocols are being used to talk between the same two nodes, they can reuse a single connection rather than having two open connections